src/plugins/janus_streaming.c

```diff
@@ -3394,6 +3428,8 @@ static json_t *janus_streaming_process_synchronous_request(janus_streaming_sessi
 			json_t *scrypto = json_object_get(root, "srtpcrypto");
 			json_t *e2ee = json_object_get(root, "e2ee");
 			json_t *pd = json_object_get(root, "playoutdelay_ext");
+			json_t *abscaptime = json_object_get(root, "abscapturetime_ext");
+			json_t *abscaptime_src_id = json_object_get(root, "abscapturetime_src_ext_id");
```

Comment:
There's no check on the ID value here (<1, >14), like there is when creating the mountpoint statically. As a result, an invalid value may be passed to `janus_streaming_create_rtp_source` when creating a mountpoint dynamically.

Reply:
sure, added missing check